src/Microsoft.VisualStudio.Threading.Analyzers.CodeFixes/Microsoft.VisualStudio.Threading.Analyzers.CodeFixes.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>netstandard1.3</TargetFramework>
```

Comment:
How is this possible? I thought Roslyn targeted netstandard2.0 nowadays.

Reply:
Roslyn 3 targets netstandard2.0. Roslyn 2 targets netstandard1.3. If you want to ensure compatibility with Visual Studio 2017 clients, the build needs to target netstandard1.3.